src/app/plugins/animal.plugin.ts

```diff
@@ -0,0 +1,189 @@
+import Constants from '../../common/constants';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IHttpResponse, IMessage, Maybe } from '../../common/types';
+import { MessageEmbed } from 'discord.js';
+import ms from 'ms';
+
+export default class AnimalPlugin extends Plugin {
+  public commandName: string = 'animal';
+  public name: string = 'Animal Plugin';
+  public description: string = 'Get wild with a fraction of gods creatures';
+  public usage: string =
+    'animal <Species (optional)> <subSpecies(Optional)> | animal list (species or <species> or subSpecies)';
+  public pluginAlias = ['wild', 'species'];
+  public permission: ChannelType = ChannelType.Public;
+  public pluginChannelName: string = Constants.Channels.Public.Pets;
+
+  private _API_URL: string = 'https://api.fetchit.dev/';
+
+  private _validAnimals: Set<string> = new Set([]); // All animals and species
+  private _species: string[] = []; // All species.
+  private _subspecies: ISubSpecies[] = []; // All subspecies housed by their species (not all species show here).
+
+  private _speciesEmbed: Maybe<MessageEmbed>;
+  private _subspeciesEmbed: Maybe<MessageEmbed>;
```

Comment:
Why use class members for the embeds? They are always written over anyways

Reply:
But it isn't, It only rewrites them if they are more than a day old, as the API is currently been worked on.